src/lib/teads-aws/README.md

```diff
@@ -1,6 +1,6 @@
 # Teads' AWS Estimation Plugin
 
-> [!NOTE] > `Teads-AWS` is a community plugin, not part of the IF standard library. This means the IF core team are not closely monitoring these plugins to keep them up to date. You should do your own research before implementing them!
+[!CAUTION] **This plugin is deprecated and will be deleted, likely before July 2024. We no longer want to support this plugin. It is possibel to replicate its behaviour using a set of generic arithmetic plugins that now come bundled in IF. You should start migrating your pipelines over to this method as soon as possible instead of relying on this plugin.**
```

Comment:
```suggestion
> [!CAUTION] **This plugin is deprecated and will be deleted, likely before July 2024. We no longer want to support this plugin. It is possibel to replicate its behaviour using a set of generic arithmetic plugins that now come bundled in IF. You should start migrating your pipelines over to this method as soon as possible instead of relying on this plugin.**
```